crates/next-core/src/next_client/context.rs

```diff
@@ -128,7 +129,7 @@ pub async fn add_next_transforms_to_pages(
             ]),
         ]),
         vec![ModuleRuleEffect::AddEcmascriptTransforms(
-            EcmascriptInputTransformsVc::cell(vec![EcmascriptInputTransform::NextJs]),
+            EcmascriptInputTransformsVc::cell(vec![EcmascriptInputTransform::NextJsPageSsr]),
         )],
     ));
```

Comment:
Could you add a new custom rule here to add the next font transform, instead of adding a new property on `ModuleOptionsContext`? Similar to what we're doing with `NextJs` above.

Reply:
This should prevent transforming `@next/font` outside of the correct contexts (i.e. it shouldn't be applied in `pages/api` etc) 😄